Makefile

```diff
@@ -53,12 +53,11 @@ package: lib-native lib-native-release
 		LIBDIR=$(TARGET_DIR)/$$RELEASE; \
 		rm -r -f dist/$$ARCHIVEDIR; \
 		rm -f dist/$$ARCHIVEFILE; \
-		mkdir dist/$$ARCHIVEDIR; \
-		mkdir dist/$$ARCHIVEDIR/include; \
-		mkdir dist/$$ARCHIVEDIR/lib; \
+		mkdir -p dist/$$ARCHIVEDIR/include/webgpu; \
+		mkdir -p dist/$$ARCHIVEDIR/lib; \
 		cp ./dist/wgpu-native-git-tag                 dist/$$ARCHIVEDIR; \
-		cp ./ffi/webgpu-headers/webgpu.h              dist/$$ARCHIVEDIR/include; \
-		cp ./ffi/wgpu.h                               dist/$$ARCHIVEDIR/include; \
+		cp ./ffi/webgpu-headers/webgpu.h              dist/$$ARCHIVEDIR/include/webgpu; \
+		cp ./ffi/wgpu.h                               dist/$$ARCHIVEDIR/include/webgpu; \
```

Comment:
Does `wgpu.h` need to be under `include/webgpu` too?

Reply:
I feel this should be `include/wgpu.h` or `include/wgpu/wgpu.h` or `include/wgpu-native/wgpu.h`.
I'd prefer the first one.

Reply:
I really don't know what would be best practice here. @Sahnvour what do you think?